File: HELLO_PYTHON/day06/mysql_delete.py
import pymysql
from sqlalchemy.dialects.mysql.pymysql import MySQLDialect_pymysql
 
# MySQL에 연결
conn = pymysql.connect(host='localhost', user='root', password='python',
                       db='python', port=3305, charset='utf8')

e_id = "5"

curs = conn.cursor()

# 수정된 SQL 쿼리에는 각 변수 주변에 따옴표(')를 추가했습니다.
sql = f"""
    delete from emp
    where e_id='{e_id}'   
"""


cnt = curs.execute(sql)
print(cnt)
#print(curs.rowcount)


conn.commit()

curs.close()
conn.close()             

File: HELLO_NODE2/mysql_delete.js
const mysql      = require('mysql'); //import 역할
const connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'root',
  password : 'python',
  database : 'python',
  port : 3305 //기본 포트가 3306이므로 포트 지정도 해주어야 한다.
});


connection.connect(function(err) {
  var sql = "DELETE FROM emp WHERE e_id = '4'";
  connection.query(sql, function (err, result) {
    console.log("1 record deleted");
	console.log("Number of records deleted: " + result.affectedRows);
  });
});



